### PR TITLE
Feature/order product category

### PR DESCRIPTION
**Pull request by Sakshi-Prajapat**

### CODE REVIEW

File: admin/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+
+function App() {
+  return (
+    <>
+      <Header />
+      <Box display="flex">
+        <SideBar />
+        <Box width="100%" height="100vh" overflow="auto">
+          {/* <Dashboard /> */}
```
Comment (tw-dev-alfaiz) [inline admin/src/App.tsx]: Please remove commented code.

File: admin/src/layout/sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      name: "Inbox",
+      icon: <MessageOutlinedIcon />,
+      path: "/inbox",
+    },
+  ];
+
+  return (
+    <>
+      <SidebarBox>
+        {/* <DashboardBox>
```
Comment (tw-dev-alfaiz) [inline admin/src/layout/sidebar/Sidebar.tsx]: Remove unnecessary commented code.

File: admin/src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+          {/* <Orders/> */}
+        {/* <Products/> */}
+        {/* <Category/> */}
```
Comment (tw-vineet) [inline admin/src/App.tsx]: why are you not removing the commented code

File: admin/src/screen/category/Category.tsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+          <Typography fontWeight="700" fontSize="24px">
+            Category
+          </Typography>
+          <Button variant="contained">+ Add Category</Button>
+        </Box>
+        <Grid2 container spacing={4} gap={2}>
+          {category.map((item) => {
+            return (
+              <Grid2 size={{ lg: 4 }}>
+                <Card sx={{ width: "350px", height: "324px" }}>
```
Comment (tw-vineet) [inline admin/src/screen/category/Category.tsx]: why sx is using so many times ?

File: admin/src/screen/category/style.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import { Box, Grid2 , Button } from "@mui/material";
+import styled from "styled-components";
+
+export const CategoryBox = styled(Box)({
+  backgroundColor:"#F5F6FA",
+  padding:"30px",
+  // height:"100vh",
```
Comment (tw-vineet) [inline admin/src/screen/category/style.ts]: remove it

File: admin/src/screen/orders/Orders.tsx [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+    <>
+      <OrderBox>
+        <Box display="flex" justifyContent="space-between">
+          <Typography fontWeight="700" fontSize="24px">
+            Orders
+          </Typography>
+          <Box display="flex" gap={2}>
+            <Button
+              variant="outlined"
+              style={{
```
Comment (tw-vineet) [inline admin/src/screen/orders/Orders.tsx]: why style in mui  ??

File: admin/src/screen/orders/Orders.tsx [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+          <Box>
+            <TextField
+              select
+              defaultValue="EUR"
+              size="small"
+              slotProps={{}}
+            ></TextField>
+          </Box>
+          <Box display="flex" gap={2}>
+            <Box
```
Comment (tw-vineet) [inline admin/src/screen/orders/Orders.tsx]: make styled component

File: admin/src/screen/orders/Orders.tsx [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+              display="flex"
+              justifyContent="center"
+              alignItems="center"
+              color="#1E5EFF"
+              bgcolor="white"
+              borderRadius="4px"
+            >
+              <BorderColorIcon />
+            </Box>
+            <Box
```
Comment (tw-vineet) [inline admin/src/screen/orders/Orders.tsx]: make styled component

File: admin/src/screen/orders/Orders.tsx [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+            >
+              <DeleteIcon />
+            </Box>
+          </Box>
+        </Box>
+        <DataGrid
+          rows={row}
+          columns={column}
+          pageSizeOptions={[5, 10]}
+          sx={{ border: 0 }}
```
Comment (tw-vineet) [inline admin/src/screen/orders/Orders.tsx]: why sx ?

File: admin/src/App.tsx (hunk 1 above)
Comment (Sakshi-Prajapat) [reply admin/src/App.tsx]: yes done

File: admin/src/layout/sidebar/Sidebar.tsx (hunk 2 above)
Comment (Sakshi-Prajapat) [reply admin/src/layout/sidebar/Sidebar.tsx]: yes removed

Comment (Sakshi-Prajapat) [reply admin/src/layout/sidebar/Sidebar.tsx]: yes removed

**Pull request merged**